Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Controllers/Api/ExportController.cs

```diff
@@ -78,10 +82,14 @@ public ActionResult<IExportProvider[]> GetExportProviders()
         [HttpPost]
         [Route("data")]
         [Authorize(ModuleConstants.Security.Permissions.Access)]
-        public ActionResult<ViewableSearchResult> GetData([FromBody]ExportDataRequest request)
+        public async Task<ActionResult<ViewableSearchResult>> GetData([FromBody]ExportDataRequest request)
         {
-            var currentUserName = _userNameResolver.GetCurrentUserName();
-            request.DataQuery.UserName = currentUserName;
+
+            var authorizationResult = await _authorizationService.AuthorizeAsync(User, null, request.ExportTypeName + "ExportDataPolicy");
```

Comment:
Could we pass request as resource here, to make possibility to limit the results?